chapter-06/inprogress/suggestions.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import "sort"
+
+// bookSuggestions are the suggested books for a given book.
```

Comment:
Usually these are called recommendations, because these are commonly referred to as "recommender systems" in literature. e.g. https://en.wikipedia.org/wiki/Recommender_system

But, suggestion can be also fine.

Reply:
Replaced with recommendation everywhere.